data/50-default.json

```diff
@@ -128,5 +128,12 @@
     "ref-kind": "app",
     "name": "com.endlessm.Clubhouse",
     "branch": "stable"
+  },
+  {
+    "action": "uninstall",
+    "serial": 2019102500,
```

Comment:
Interesting choice of date – 25th october was several weeks ago! It doesn't matter of course, it's just a monotonically-increasing integer.